chia/server/reconnect_task.py

```diff
@@ -1,29 +0,0 @@
-from __future__ import annotations
-
-import asyncio
-from logging import Logger
-
-from chia.server.server import ChiaServer
-from chia.types.peer_info import PeerInfo
-
-
-def start_reconnect_task(server: ChiaServer, peer_info: PeerInfo, log: Logger) -> asyncio.Task[None]:
```

Comment:
nothing this function accesses from `server` starts with an underscore. I would think `all_connections` and `start_client()` are meant to be public